tests/core/training/test_interactive.py

```diff
@@ -733,6 +737,51 @@ def test_retry_on_error_success(monkeypatch: MonkeyPatch):
     m.assert_called_once_with("export_path", 1, a=2)
 
 
+@pytest.mark.parametrize(
+    "action_name, question",
+    [
+        (
+            "action_unlikely_intent",
+            f"The bot wants to run 'action_unlikely_intent' "
+            f"to indicate that the last user message was unexpected "
+            f"at this point in the conversation. "
+            f"Check out IntentTEDPolicy ({DOCS_URL_POLICIES}/#intent-ted-policy) "
+            f"to learn more. Press any key to continue…",
+        ),
+        ("action_other", "The bot wants to run 'action_other', correct?"),
+    ],
+)
+async def test_correct_question_for_action_name_was_asked(
+    monkeypatch: MonkeyPatch,
+    mock_endpoint: EndpointConfig,
+    action_name: Text,
+    question: Text,
+):
+    # mock those functions that shouldn't be called
+    monkeypatch.setattr(
+        interactive,
+        "retrieve_tracker",
+        asyncio.coroutine(
```

Comment:
asyncio.coroutine is deprecated as of Python 3.8 - can we please us `async def` instead?